app/services/ClubManagementService.php

```diff
@@ -33,9 +33,9 @@ static function clubExists($slug)
         return file_exists(club_data_path($slug));
     }
 
-    function getClub(): Club
+    function getClub()
     {
-        $results = $this->db->em()->createQuery("SELECT c FROM Club c")->getResult();
+        $results = em()->createQuery("SELECT c FROM Club c")->getResult();
```

Comment:
I don't think you understood the value of not using the singleton here.

If you do this, it will be a huge migration if / when we try to migrate to a multithreaded PHP runtime like workerman... or unit tests.

I think we should actually start testing our code way more than we do. It would ensure we follow best practices when it comes to not using singletons all over the place.

Reply:
Overall the PR is ok but I would like you to go back to using the instance of the db here :)